src/app/components/Form/QualificationsChecklist.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { Checkbox } from 'react-bootstrap';
+import { default as FormFieldWrapper, FormFieldWrapperProps } from './FormFieldWrapper';
+
+
+export default class QualificationsChecklist extends React.PureComponent<FormFieldWrapperProps>{
+    render(){
+        const {input:{value, onChange}} = this.props;
+        return (
+            <FormFieldWrapper {...this.props}>
+                <Checkbox onChange={onChange} value={value}>
+                    Transfers
+                </Checkbox>
+                {' '}
```

Comment:
I think in order to improve cross browser viewing/support we might want to either place these inside a div that arranges them vertically or use `<br/>` for line breaks, or even `<ul><li></li></ul>` perhaps, or I'm not sure if there is something within bootstrap that will help with this.  I'm just not sure that all browsers will treat the `{' '}` the same way

Reply:
replaced {' '} with <br/>